src/Security/Authorization/Policy/src/AuthorizationEndpointConventionBuilderExtensions.cs

```diff
@@ -166,6 +171,7 @@ private static void RequireAuthorizationCore<TBuilder>(TBuilder builder, IEnumer
             {
                 endpointBuilder.Metadata.Add(data);
             }
+            endpointBuilder.Metadata.Add(new AuthorizationPolicyCache());
```

Comment:
Wondering why there's no check for existence `AuthorizationPolicyCache` in the metadata the same as its done in `RequirePolicyCore`.